signal_analog/flow.py

```diff
@@ -379,6 +379,8 @@ def __str__(self):
             str_arg = str(self.arg)
         return "%s=%s" % (self.keyword, str_arg)
 
+    def __eq__(self, other):
```

Comment:
This is only used of testing.  Do we want this?

Reply:
I don't see why not.